sdk/metricsadvisor/ai-metrics-advisor/samples/javascript/README.md

```diff
@@ -1,3 +1,13 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-cognitive-services
```

Comment:
shouldnt the products list include the metrics advisor?

Reply:
It's just been discussed this morning,  Kate mentioned that the metrics advisor product is not ready/valid yet so we will use this for now.  It will be updated in the future

https://review.docs.microsoft.com/en-us/new-hope/information-architecture/metadata/taxonomies?branch=master#dev-lang

Reply:
fair enough